Sources/JobsRedis/RedisJobQueue.swift

```diff
@@ -25,30 +26,43 @@ public final class RedisJobQueue: JobQueueDriver {
     public struct JobID: Sendable, CustomStringConvertible {
         let id: String
 
-        public init() {
+        let delayUntil: Int
+
+        public init(delayUntil: Int?) {
             self.id = UUID().uuidString
+            self.delayUntil = delayUntil ?? 0
         }
 
         /// Initialize JobID from String
         /// - Parameter value: string value
         public init(_ value: String) {
-            self.id = value
+            let parts = value.components(separatedBy: ":")
+            self.id = parts[0]
+            self.delayUntil = if parts.count > 0 {
+                Int(parts[1]) ?? 0
+            } else {
+                0
+            }
         }
 
         public init(from decoder: Decoder) throws {
             let container = try decoder.singleValueContainer()
```

Comment:
Do we need the `init(from:)` and `encode(to:` anymore

Reply:
no longer needed